### PR TITLE
Add real ip and proxy module

### PR DESCRIPTION
**Pull request by pawurb**
Hi. Could you enable those 2 modules? They are needed to enable passing real IP for apps behind Cloudflare proxy as described here: https://www.wpintense.com/2017/03/24/configuring-nginx-pass-real-ip-addresses-cloudflare-compatible-fail2ban-wordpress/ .

I'm using it with success with my fork now.

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -101,7 +101,6 @@ if [[ ! -f "${CACHE_DIR}/bin/nginx" ]]; then
     --without-http_map_module \
     --without-http_split_clients_module \
     --without-http_referer_module \
-    --without-http_proxy_module \
```
Comment (josegonzalez) [inline bin/compile:104]: Why this change?

Comment (pawurb) [reply bin/compile:104]: It's needed to use `proxy_set_header`.

**Comment by pawurb** [comment]
@josegonzalez any chance to get it merged ?
